pages/contact-representative.js

```diff
@@ -22,7 +22,7 @@ const ContactRepresentative = (props) => {
         onHandleErrorClick={OnHandleErrorClick}
       />
       <Heading as="h1" size="LARGE">
-        {props.t("Operator contact details")}
+        {props.t("Contact Details")}
```

Comment:
Would suggest changing this to 'Contact representative details' as users could assume it means establishment details and enter the wrong information.